docs/settingUp.md

```diff
@@ -32,12 +40,18 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
+   
+ **Verification** : Check the gradle folder for the existence of the SDK.The folder path will be like this : 
```

Comment:
Check your Gradle folder (the directory can be found with the command `./gradlew printUserHomeDir`). A folder `appengine-sdk` should be present.